### PR TITLE
Central

### PR DESCRIPTION
**Pull request by nelealbers**
Fixes https://github.com/PerfectFit-project/virtual-coach-issues/issues/266. 

Changes:
- rule to detect "urgent" intent at start of conversation and then send a message for people to contact a human (need to refine what this message is)
- when we get a message with low nlu confidence at start of conversation, we present the central mode menu.

### CODE REVIEW

**Comment by nelealbers** [comment]
@wbaccinelli From what I can see, the failed Python CI test has nothing to do with what I have done. For the Rasa CI, does it fail because we have now split up the files?

**Comment by nelealbers** [comment]
@wbaccinelli Thanks, fixed!

File: Rasa_Bot/data/rules.yml [hunk 1]
```diff
@@ -1,10 +1,21 @@
...
+  - action: action_end_dialog
+
+# Only applies at start of conversation
+- rule: Process when people talk about urgent things that we cannot handle
+  conversation_start: true
+  steps:
+  - intent: urgent
+  - action: utter_refer
+  - action: utter_central_mode_options
+  - action: action_end_dialog
```
Comment (wbaccinelli) [inline Rasa_Bot/data/rules.yml:18]: I tried this, but the confidence level of confidence is too low (even if I write precisely one of the examples). I think we have to pay a lot of attention on how we train the model. Maybe we should have different intents for different words?

**Review by wbaccinelli — APPROVED** [review]
This is good. We need the content to be fully defined, to test that everything is working properly

File: Rasa_Bot/data/rules.yml [hunk 2]
```diff
@@ -1,10 +1,21 @@
...
+- rule: Give user central mode options when they send a message with low NLU confidence
   steps:
   - intent: nlu_fallback
   - action: utter_default
+  - action: utter_central_mode_options
+  - action: action_end_dialog
+
+# Only applies at start of conversation
+- rule: Process when people talk about urgent things that we cannot handle
+  conversation_start: true
```
[the remark this answers is not in the document]
Comment (nelealbers) [reply Rasa_Bot/data/rules.yml:13]: @wbaccinelli Ah good point. I will adapt the action_end_dialog action.

**Comment by nelealbers** [comment]
@wbaccinelli I now got things to work. There was a but that sometimes made things break that was fixed in 3.2.8: https://rasa.com/docs/rasa/changelog. So I moved everything to 3.2.8. Let me know if I should merge this now.

File: Rasa_Bot/actions/Dockerfile [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # Dockerfile for rasa actions server
 
 # Pull SDK image as base image, version must be compatible with Rasa version
-FROM rasa/rasa-sdk:3.1.1
+FROM rasa/rasa-sdk:3.2.1
```
[the remark this answers is not in the document]
Comment (wbaccinelli) [reply Rasa_Bot/actions/Dockerfile:4]: @nelealbers I think this needs to be changed, but everything is working now! Nice work!

[the remark this answers is not in the document]
Comment (nelealbers) [reply Rasa_Bot/actions/Dockerfile:4]: @wbaccinelli I thought so at first too, but it seems that rasa-sdk has different non-matching version numbers: https://rasa.com/docs/action-server/rasa-sdk-changelog/. There newest version is 3.2.1 according to their changelog.

[the remark this answers is not in the document]
Comment (wbaccinelli) [reply Rasa_Bot/actions/Dockerfile:4]: @nelealbers ok, perfect! I will merge the branch then.

**Pull request merged**